src/app/vendors/[id]/milestones/page.tsx

```
@@ -0,0 +1,94 @@
+                .eq('id', milestoneId)
+                .single();
+
+            if (!error && data) {
+                setMilestone(data);
+            }
+            setIsLoading(false);
+        }
+        fetchMilestone();
+    }, [milestoneId]);
+
+    if (isLoading || !milestone) {
```

ESLint

[error] Delete `··`

(prettier/prettier)